### PR TITLE
auto

### PR DESCRIPTION
**Pull request by vkedia05**

### CODE REVIEW

File: src/main/java/org/bitbuckets/odometry/OdometryControl.java [hunk 1]
```diff
@@ -31,28 +27,18 @@ public class OdometryControl implements IOdometryControl {
...
-        Optional<Pose3d> res = visionControl.estimateRobotPose();
-
-        if (res.isPresent()) {
-            Pose2d realPose = res.get().toPose2d();
-
-            swerveDrivePoseEstimator.addVisionMeasurement(realPose, epoch);
-
-        } else {
-
-        }
```
Comment (cpostbitbuckets) [inline src/main/java/org/bitbuckets/odometry/OdometryControl.java:47]: Why was this deleted?

File: src/main/java/org/bitbuckets/odometry/OdometryControl.java [hunk 2]
```diff
@@ -31,28 +27,18 @@ public class OdometryControl implements IOdometryControl {
...
 
     public void logLoop() {
-        debuggable.log("yaw", pigeonIMU.getYaw());
-        debuggable.log("pitch", pigeonIMU.getPitch());
-        debuggable.log("roll", pigeonIMU.getRoll());
-        debuggable.log("heading", pigeonIMU.getAbsoluteCompassHeading());
+        debuggable.log("yaw", (pigeonIMU.getYaw()));
+        debuggable.log("pitch", (pigeonIMU.getPitch()));
+        debuggable.log("roll", (pigeonIMU.getRoll()));
+        debuggable.log("heading", pigeonIMU.getYaw());
```
Comment (cpostbitbuckets) [inline src/main/java/org/bitbuckets/odometry/OdometryControl.java:41]: Let's remove heading.

**Review by cpostbitbuckets — CHANGES_REQUESTED** [review]
Other than deleting vision code in odometry, it looks good.

Was that intentional?

File: src/main/java/org/bitbuckets/odometry/OdometryControl.java (hunk 1 above)
Comment (vkedia05) [reply src/main/java/org/bitbuckets/odometry/OdometryControl.java:47]: matt told me to, said it was not needed

**Comment by superyuuki** [comment]
Merge this
